core/options.js

```diff
@@ -250,10 +252,19 @@ Blockly.Options.prototype.getMetrics;
 Blockly.Options.parseMoveOptions_ = function(options, hasCategories) {
   var move = options['move'] || {};
   var moveOptions = {};
-  if (move['scrollbars'] === undefined && options['scrollbars'] === undefined) {
+  if (move['hasVerticalScrollbars'] !== undefined || move['hasHorizontalScrollbars'] !== undefined) {
+    moveOptions.hasVerticalScrollbars = !!move['hasVerticalScrollbars'];
```

Comment:
In the interest of keeping the options consistent, let's not have `has` in the name of the option. (e.g.: to determine whether or not the wheel is enabled, we just use the term `wheel` rather than `hasWheel`).
Another thing to consider is splitting this into an object such that: 
- if scrollbars is a boolean and true, enable both
- if scrollbar is an object, look further for `horizontal` and `vertical` fields to determine if each axis is enabled.
The resulting interface would look something like this:

```
interface MoveOptions {
   drag: boolean;
   scrollbars: boolean | ScrollbarOptions;
   wheel: boolean
}

interface ScrollbarOptions {
    horizontal: boolean;
    vertical: boolean;
}
```

Reply:
Done.